system-tests.js

```diff
@@ -125,7 +125,7 @@ process.chdir(workingDirectoryPath)
 
 // api
 run(`ern create-api ${apiName} --skipNpmCheck`)
-const apiPath = path.join(process.cwd(), `react-native-${apiName}-api`)
+const apiPath = path.join(process.cwd(), `${apiName}`)
```

Comment:
minor : can remove the template literal string here and just use the variable directly